arviz/stats/stats_utils.py

```diff
@@ -427,6 +427,10 @@ def get_log_likelihood(idata, var_name=None):
 
 BASE_FMT = """Computed from {{n_samples}} by {{n_points}} log-likelihood matrix
 
+The scale is now log by default. Use 'scale' argument if you rely on a specific value.
+A higher log-score (or a lower deviance) indicates a model with better predictive 
+accuracy.

```

Comment:
I would put this on the bottom of the message (and if possible even below pointwise summary). I would then be able to copy/screenshot the upper part of the printed text and omit the warning

Reply:
Good idea, will update